Playmate/View Controllers/Session Details View/SessionDetailsViewController.m

```diff
@@ -33,11 +34,14 @@ @interface SessionDetailsViewController () <UICollectionViewDelegate, UICollecti
 @implementation SessionDetailsViewController
 
 PFUser *me;
+BOOL partOfSession;
```

Comment:
devs name booleans as if the were asking questions for example here:
`isPartOfSession`